components/board/board.go

```diff
@@ -107,15 +107,3 @@ func FromRobot(r robot.Robot, name string) (Board, error) {
 func NamesFromRobot(r robot.Robot) []string {
 	return robot.NamesByAPI(r, API)
 }
-
-// RemoveCallbacks removes the callbacks from the given interrupts.
```

Comment:
I removed intentionally, i think its cleaner now to just call `RemoveCallback` on the interrupts.

Reply:
we're removing those from the interface completely though, as per the scope doc we can still use them if a driver uses the pinwrapper library, but not in server.go.

If you'd like, you can have the helper take in a digital interrupt for the server side implementation.

Reply:
oh wait I see your comment about that above now, we won't be removing the `removeCallback` interrupt function completely it just will be a helper instead of being on the interface.